src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24414.cs

```diff
@@ -16,8 +16,34 @@ public Issue24414(TestDevice device)
 		[Category(UITestCategories.Visual)]
 		public void Issue24414Test()
```

Comment:
On Android, could use https://github.com/dotnet/maui/blob/daf88668adc3479c15c03a87d8e320ce5f89503f/src/TestUtils/src/UITest.Appium/HelperExtensions.cs#L1771 to get memory info and validate never exceed a value, and avoid detecting degradation in memory consumption due to any change